src/main/resources/cli_en_US.properties

```diff
@@ -9,6 +9,7 @@ error_msg=Error: %s
 help.option.help=Print help information
 help.option.debug=Enable debug logging
 help.option.verbose=Enable verbose logging
+help.option.no-color=Disable color in terminal logging
```

Comment:
@infotexture is the copy ok?

Reply:
I think that's fine. 👍 

— “…in terminal _output_” might be an alternative, but not sure it’s better 🤔

Reply:
@infotexture I'm ok with either, you choose.

Reply:
Not sure this any better or of any help, but what about "Disable color in terminal"?